app/configs/muiThemeProvider.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import { useEffect } from "react";
+import { ThemeProvider } from "@mui/material/styles";
+import theme from "./theme";
+
+const MuiThemeProvider = ({ children }: { children: React.ReactNode }) => {
```

Comment:
no need to `React.ReactNode` since already import React in line 3
```
import { ReactNode } from "react";

.... { children:ReactNode }
```

Reply:
done